File: support_tools/swigwin-3.0.12/Examples/test-suite/python/constover_runme.py
import constover
import sys
error = 0

p = constover.test("test")
if p != "test":
    print "test failed!"
    error = 1

p = constover.test_pconst("test")
if p != "test_pconst":
    print "test_pconst failed!"
    error = 1

f = constover.Foo()
p = f.test("test")
if p != "test":
    print "member-test failed!"
    error = 1

p = f.test_pconst("test")
if p != "test_pconst":
    print "member-test_pconst failed!"
    error = 1

p = f.test_constm("test")
if p != "test_constmethod":
    print "member-test_constm failed!"
    error = 1

p = f.test_pconstm("test")
if p != "test_pconstmethod":
    print "member-test_pconstm failed!"
    error = 1

sys.exit(error)

File: support_tools/swigwin-3.0.12/Examples/test-suite/javascript/constover_runme.js
var constover = require("constover");

p = constover.test("test");
if (p != "test") {
    throw "test failed!";
}

p = constover.test_pconst("test");
if (p != "test_pconst") {
    throw "test_pconst failed!";
}

f = new constover.Foo();

p = f.test("test");
if (p != "test") {
    throw "member-test failed!";
}

p = f.test_pconst("test");
if (p != "test_pconst") {
    throw "member-test_pconst failed!";
}

p = f.test_constm("test");
if (p != "test_constmethod") {
    throw "member-test_constm failed!";
}

p = f.test_pconstm("test");
if (p != "test_pconstmethod") {
    throw "member-test_pconstm failed!";
}
